src/base/preferences.cpp

```diff
@@ -761,6 +761,26 @@ void Preferences::setWebUICustomHTTPHeaders(const QString &headers)
     setValue("Preferences/WebUI/CustomHTTPHeaders", headers);
 }
 
+bool Preferences::isWebUIReverseProxyEnabled() const
+{
+    return value("Preferences/WebUI/ReverseProxyEnabled", false).toBool();
```

Comment:
IMO, "reverse proxy enabled" confuses. One may think that there is some built-in "reverse proxy" feature.
Maybe "forwarded client IP enabled/used/allowed"?

Reply:
Maybe "Reverse proxy support" ? I think the word "reverse proxy" makes the purpose clear.